ntoskrnl/mm/balance.c

```diff
@@ -380,7 +382,7 @@ MiBalancerThread(PVOID Unused)
                 ULONG OldTarget = InitialTarget;
 
                 /* Trim each consumer */
-                for (i = 0; i < MC_MAXIMUM; i++)
+                for (ULONG i = 0; i < MC_MAXIMUM; i++)
```

Comment:
Change of dubious value ^^

Reply:
What is the "dubious value"? Do you mean `MC_MAXIMUM`?

Reply:
The `ULONG i` variable declaration move.

Reply:
Is there any problem with it?

Reply:
I don't see any problem with it. Other places in Mm already use this style.
Like this:
[https://git.reactos.org/?p=reactos.git;a=blob;f=ntoskrnl/mm/section.c;h=73b414315fe38fef6fc4e1b7c8968a6797e5f343;hb=1faa571f3f0445f46554ebe33bf105a87baeea2e#l4415](https://git.reactos.org/?p=reactos.git;a=blob;f=ntoskrnl/mm/section.c;h=73b414315fe38fef6fc4e1b7c8968a6797e5f343;hb=1faa571f3f0445f46554ebe33bf105a87baeea2e#l4415)